web/src/components/atoms/ContentSection/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
+import { ReactNode } from "react";
+
+export type Props = {
+  title: string;
+  danger?: boolean;
+  extra?: ReactNode;
```

Comment:
What is the extra for?

Reply:
@KaWaite , inspired by `extra` in antd design, used if you need actions/content on the other side of the title.

Reply:
If we don't need it in the design, please remove it. We can add things like that when the need arises

Reply:
@KaWaite , it is used by the `Project overview` page, that's why it is added.

Reply:
So there are a couple issues with the current implementation. First, the name just doesn't really convey the meaning of what it is. Right now I can't think of a better name, so if you can think of something more meaningful, please suggest it. 
Second, the Title inside ContentSection has no padding or margin, and the Header doesn't wrap, which means the extra prop's component isn't really going to work nicely if it is long or the browser width becomes a little narrow. 

If you can solve the above issues your approach is fine. Another option would be to make Title `string | ReactNode`, then you could make the Header fully customizable.